models/stg_intercom__admin.sql

```diff
@@ -35,5 +35,4 @@ final as (
 )
 
 select * 
-from final
-where not coalesce(_fivetran_deleted, false)
```

Comment:
Removing this is not helpful for analytics - it means you have missing admin data when trying to reconcile with historical conversations. Leaving this in is more helpful

Reply:
Thanks for providing this insight @robertpknight, I see how including deleted admins would be helpful for reconciliation with historical conversations. With this understanding, I would imagine retaining deletes from other tables in the Intercom package would be beneficial for historical reference. The other tables in the Intercom packages that capture deletes are `company_history`, `company_tag_history`, `tag`, `team_admin`, `team`.  

Do you believe retaining deleted records for the mentioned tables would be beneficial for your analysis?